### PR TITLE
PLA-3888: fixed trouble with element alignment

### PR DESCRIPTION
**Pull request by ariyDez**

### CODE REVIEW

**Comment by enigma91** [comment]
What is the task number? It seems that branch was created with wrong task number

**Comment by ariyDez** [comment]
I am mistaken. It is PLA-3888

File: css/audit.css [hunk 1]
```diff
@@ -519,6 +520,10 @@ html, body {
     color: #4c4c4d;
 }
 
+.violation-name {
+    width: 500px;
```
Comment (enigma91) [inline css/audit.css]: Have you checked this solution on a small screens? Because this is not the way which is described in a task

**Comment by george-moberly** [comment]
Please complete reviews post-merge

**Pull request merged**